### PR TITLE
My modifications on the train-test-split branch

### PR DESCRIPTION
**Pull request by StephenChan**

### CODE REVIEW

File: classify_allsource_s3_fixed_labels.py [hunk 1]
```diff
@@ -43,6 +42,17 @@ def log_memory_usage(message):
...
 
+
+OUTPUT_BUCKET = 'coralnet-mermaid-share'
+OUTPUT_PATH = 'allsource'
+
+
+def write_bytestream_from_s3(output_storage, s3_filepath, out_stream):
+    in_stream = output_storage.load(s3_filepath)
+    in_stream.seek(0)
+    shutil.copyfileobj(in_stream, out_stream)
```
Comment (yeelauren) [inline classify_allsource_s3_fixed_labels.py:53]: @StephenChan can you ELI5 what this does differently from the other s3 writing operations?  I

Comment (StephenChan) [reply classify_allsource_s3_fixed_labels.py:53]: Hmm, I copied this function from a script I'd written before, but honestly not sure why I wrote it this way. Maybe it made a little more sense in an earlier pyspacer version, or maybe not... but, all it's doing is copying a file. In my usages specifically, copying from S3 to filesystem.

Here's what probably could have been done in place of the `write_bytestream_from_s3()` usages:

```python
model_loc_s3 = DataLocation("s3", bucket_name=OUTPUT_BUCKET, key=classifier_filepath)
model_loc_fs = DataLocation("filesystem", key=classifier_filename)
model_loc_fs.store(model_loc_s3.load())
# And similarly for valresult
```

One thought is that maybe using a Storage instance instead of DataLocations might've been convenient since I was doing a delete() later (and only Storage has a delete() method). But that doesn't justify everything about this function I wrote.
